classes_assignment.js

```diff
@@ -0,0 +1,61 @@
+// class Human {
+//   constructor(name, age){
+//     this.name = name;
+//     this.age = age;
+//   }
+// ageOneYear() {
+//   return this.age++
+// }
+//
+// eating(str) {
+//   return "mmm, mmm, mmm, I\'m love\'n it"
+// }
+// }
+//
+// let jrJ = new Human("Jung Rae Jang", 28)
+// console.log(jrJ.age)
+// console.log(jrJ.ageOneYear())
+// console.log(jrJ.age)
+// console.log(jrJ.eating("Tortas"))
```

Comment:
And this way, the invoke the function we don't need to wrap it in a console.log statement